### PR TITLE
Edit Game Session

### PR DESCRIPTION
**Pull request by 100kV**
Fixes: https://github.com/codepath-snv/gamenights/issues/35

### CODE REVIEW

File: Gamenights/GameDetailViewController.swift [hunk 1]
```diff
@@ -26,6 +26,12 @@ class GameDetailViewController: UIViewController, UITableViewDataSource, UITable
         
         tableView.reloadData()
     }
+    
+    override func viewWillAppear(animated: Bool) {
+        if game != nil {
```
Comment (6zz) [inline Gamenights/GameDetailViewController.swift:31]: you could also do the following to test for nil and unwind optionals in one go

```
if let game = game {
  fetchGameSessions(game.objectId)
}
```

Comment (100kV) [inline Gamenights/GameDetailViewController.swift:31]: Thanks!
